tests/Unit/Models/PasswordResetTokenTest.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace Tests\Unit\Models;
+
+use App\Models\PasswordResetToken;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Tests\TestCase;
+
+class PasswordResetTokenTest extends TestCase
+{
+    use RefreshDatabase;
+
+    public function testIsExpireWhenFalse()
+    {
+        $passwordResetToken = PasswordResetToken::factory()->createOneQuietly();
+        $this->assertNotTrue($passwordResetToken->isExpire());
+    }
+
+    public function testIsExpireWhenTrue()
+    {
+        $passwordResetToken = PasswordResetToken::factory()->createOneQuietly();
+        $passwordResetToken->update(['created_at' => now()->startOfDay()]);
```

Comment:
this test will fail if we run it 00:01 AM?

You can generate an expired token by subtracting time, e.g. `$date->sub('hour', 3)`